README.md

```diff
@@ -181,6 +184,29 @@ Note that our CI system checks that code formatting is consistent, and the build
 
 Also note that we run some `clang-tidy` linting over the code as well (see `.clang-tidy` for the list of rules enforced).  You can run `./scripts/tidy.sh` over the code before committing to ensure you haven't added any of the common problems we check for (Note: `./scripts/tidy.sh` requires the exitence of a `compile_commands.json` database.  You can generate this file by running `bear make` instead of just `make`.  The `bear` tool is installable on Ubuntu-based systems with `apt-get install bear`, and on macOS with `brew install bear`).
 
+Using the Node.js Bindings
+--------------------------
+
+The Node.js bindings are still under construction. We are working on building binaries for as many environments as possible, but they may not all be available yet. The first functionality that we are exposing is `route`, but we plan on exposing more of the tyr actor functions. Right now, the input and the output are both strings - THAT WILL CHANGE. We plan on ingesting and producing protobufs.
```

Comment:
Wondering whether we should remove the part about `tyr actor function`.

Looking at the example of how to use the node binding, it is pretty opaque to the user what `tyr` is